config.json

```diff
@@ -1,5 +1,6 @@
 {
-  "secret": "secret",
```

Comment:
Let's use secret for now - I will add `token_hmac_secret_key ` option in future commits